ios/ARgo.xcodeproj/project.pbxproj

```diff
@@ -345,8 +355,6 @@
 		F99513A01FC428550009D2B6 /* FBSDKCoreKit.framework */ = {isa = PBXFileReference; lastKnownFileType = wrapper.framework; path = FBSDKCoreKit.framework; sourceTree = "<group>"; };
 		F99513A11FC428560009D2B6 /* FBSDKShareKit.framework */ = {isa = PBXFileReference; lastKnownFileType = wrapper.framework; path = FBSDKShareKit.framework; sourceTree = "<group>"; };
 		F99513A21FC428560009D2B6 /* FBSDKLoginKit.framework */ = {isa = PBXFileReference; lastKnownFileType = wrapper.framework; path = FBSDKLoginKit.framework; sourceTree = "<group>"; };
-		0DF48324914C44ECA491DB66 /* AirMaps.xcodeproj */ = {isa = PBXFileReference; name = "AirMaps.xcodeproj"; path = "../node_modules/react-native-maps/lib/ios/AirMaps.xcodeproj"; sourceTree = "<group>"; fileEncoding = undefined; lastKnownFileType = wrapper.pb-project; explicitFileType = undefined; includeInIndex = 0; };
```

Comment:
@dlowkeen and @jmsjtu - is it okay to take this out?

Reply:
I have no idea... I didn't even know i did that. It may have happened when I deleted the node modules and reinstalled them. Was this not included in the json.package file?